src/core/internal/parseoptions.d

```diff
@@ -94,7 +94,24 @@ bool parseOptions(CFG)(ref CFG cfg, string opt)
                 static if (!is(typeof(__traits(getMember, cfg, field)) == function))
                 {
                     case field:
-                        if (!parse(name, tail, __traits(getMember, cfg, field), errName))
+                        alias fieldT = typeof(__traits(getMember, cfg, field));
+
+                        static if (is(fieldT == size_t))
+                        {
+                             // names of field what can contain B/K/M suffixes
+                            bool isSuffValue = (
+                                name == "initReserve" ||
+                                name == "minPoolSize" ||
+                                name == "maxPoolSize" ||
+                                name == "incPoolSize"
+                            );
```

Comment:
This seems to be unused?

Reply:
Used while parsing integers into `Config` (`CFG`) structure

Reply:
How? This is the only occurence of `isSuffValue`.

Reply:
Can you explain more?

Reply:
isSuffValue is set but never used. I assume you want to pass it as a parameter for `parseSuff` instead of `true`?

Reply:
E.g. `--DRT-gcopt=parallel:16G` should issue an error that the suffix is not supported.

Reply:
Ah, yes, thanks! Forgot to replace after debugging. Now I write patch with using of UDA, so, this code will be fixed